README.md

```diff
@@ -365,3 +365,27 @@ Done
 #### ncp-ll64
 
 Return the Link Local 64-bit IPv6 address for the node.
+
+
+## Vendor package
+
+Extension of spinel-cli by specific vendor properties and commands.
+Idea of creating vendor package is to create specific commands without interrupting in spinel core.
+Vendor module contain:
+
+| MODULE        |    DESCRIPTION                                     |
+|---------------|----------------------------------------------------|
+| vendor        | Module providing a specific vendor commands        |
+| const         | Module with constants for vendor spinel extension  |
+| codec         | Module providing a Vendor property handlers        |
+In each module is provided example how to add specific vendor codec and constant. 
+
+### Vendor commands
+
+#### help
+```bash
+spinel-cli > vendor help
+Available vendor commands:
+==============================================
+help
```

Comment:
Not sure what this means. If it is meant to replace the list of commands, can you provide the full list please? Like in the other PR? @LuDuda ?